src/modules/company/controllers/deleteAdmin.js

```diff
@@ -1,42 +1,23 @@
-import { verifyToken } from "../../../utilities/jsonWebToken.js";
+import { databasePrisma } from "../../../prismaClient.js";
 
-export const deleteAdminFromCompany = async (databasePrisma, req, res) => {
+export const deleteAdminFromCompany = async (req, res) => {
   const { admin } = req.body;
   const id = req.params.id;
+
   // Validate to see if inputs are provided correctly
   if (!id) {
     return res.status(400).send({ message: "Company id is required" });
   }
```

Comment:
Checking the id is not needed as this will have caused an error response in the company exists validation previously

```suggestion
```